config/initializers/load_libraries.rb

```diff
@@ -19,14 +19,16 @@
 require 'email_inviter'
 require 'evil_query'
 require 'federation_logger'
-require 'h_card'
 require 'hydra_wrapper'
 require 'postzord'
 require 'publisher'
 require 'pubsubhubbub'
-require 'salmon'
 require 'statistics'
 require 'stream'
-require 'webfinger'
-require 'webfinger_profile'
 require 'account_deleter'
+
+# diaspora-foundation adapters
```

Comment:
I think I was reading something by Isaac Asimov. All the blame goes on this guy.